apps/dotcom/sync-worker/src/postgres.ts

```diff
@@ -1,7 +1,7 @@
-import type {
-	PostgresSQL,
-	PostgresTransaction,
-} from '@rocicorp/zero/out/zero-pg/src/postgres-connection'
```

Comment:
They changed the type names here:
https://github.com/rocicorp/mono/pull/4251/files#diff-a4d49c645c6c1670678c52bf94fb100f31888d850a642c9effbbe17506810283